modules/apps/commerce/commerce-product-service/src/main/java/com/liferay/commerce/product/internal/layout/admin/util/CPDefinitionSitemapURLProvider.java

```diff
@@ -156,10 +152,6 @@ protected void visitLayout(
 			return;
 		}
 
-		themeDisplay = SitemapURLProviderUtil.updateThemeDisplay(
-			_language, _portal.getLocale(themeDisplay.getRequest()),
-			themeDisplay);
-
 		String currentSiteURL = _portal.getGroupFriendlyURL(
```

Comment:
why are you removing this method? this was added here https://issues.liferay.com/browse/COMMERCE-11452 and the updated themedisplay is used in this method call _portal.getGroupFriendlyURL, I'm not sure this is right to remove.

Reply:
Hi @andrea-ale-sbarra , the method is no longer necessary.

getAlternateURLs has the implementation already. 
https://github.com/liferay/liferay-portal/blob/f350a1cd720339741f646ad8041e7e0f7fec6158/portal-impl/src/com/liferay/portal/util/PortalImpl.java#L1298
https://github.com/liferay/liferay-portal/blob/f350a1cd720339741f646ad8041e7e0f7fec6158/portal-impl/src/com/liferay/portal/util/PortalImpl.java#L1403

If we update the themedisplay, we will get URLs that are the following:
"localhost:8080/en/ca/" which is not what we want. We would be essentially appending the locale twice.

Reply:
got it thanks!